dashboard/cron.py

```diff
@@ -246,18 +249,9 @@ def update_with_bq_access(self):
         # BQ Total Bytes Billed to report to status
         total_bytes_billed = 0
 
-        # the earliest start date of all courses
-        course_start_time = utils.find_earliest_start_datetime_of_courses()
-
-        # the earliest latest date of all resources_accessed
-        try:
-            latest_resource = ResourceAccess.objects.latest('access_time')
-            latest_resource_time = latest_resource.access_time
-        except ResourceAccess.DoesNotExist:
-            latest_resource_time = course_start_time
-
-        logger.info(f"Earliest Start: {course_start_time} Latest resource: {latest_resource_time}")
+        data_last_updated = Course.objects.get_data_last_updated()
 
+        status += delete_all_records_in_table("resource_access", f"WHERE access_time > %s", [data_last_updated,])
```

Comment:
So this works if `data_last_updated` is `null`, I guess?

Reply:
I guess `null` is impossible, since `update_with_bq_access` won't be invoked if `self.locked_valid_course_ids` is empty.

Reply:
Yeah, that check blocks the case of no courses and no date getting into here.